automation/ip-address-script/parameters.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Since you are not using parameters.json or template.json, delete so we don't have any unnecessary code merging.